docs/src/main/sphinx/release/release-431.md

```diff
@@ -0,0 +1,55 @@
+# Release 431 (25 Oct 2023)
+
+## General
+
+* Add support for the `CREATE OR REPLACE TABLE` statement. ({issue}`13180`)
+* Disallow a offset of `null` in the {func}`lead` and {func}`lag` functions. ({issue}`19003`)
+
+## Web UI
+
+* Mask shared secret value in logs. ({issue}`19519`)
+
+## JDBC driver
+
+* Add a new JDBC property `legacyPreparedStatements` that controls whether
```

Comment:
This is too low level. What's the user visible impact? (something like "improve query latency by reducing number of roundtrips when executing prepared statements")

Reply:
Yeah .. well.. it does do that since its off by default ..